npm prepare script run when lerna is passed --ignore-scripts

**What Renovate type are you using?**

I am using the hosted app on Github.

**Describe the bug**

4 days ago I noticed that one of the PRs opened by our renovatebot had encountered an error when running `lerna bootstrap`. It is failing at a step in our build that runs `grunt dist` as part of `npm run prepare` for one of the modules in our monorepo. This step has not changed since we began using renovatebot last August.

I also noticed that checking the box of another dependency in our master issue was not resulting in a PR being opened for quite a while. Eventually a PR was opened but it also had the same error.

**Did you see anything helpful in debug logs?**

The logs dashboard shows multiple consecutive runs with the label `lockfile-error`. The relevant portion of those logs are here:

```
DEBUG: Found compatible npm version(branch="renovate/react-helmet-6.x")
{
  "constraint": "10.15.3",
  "version": "10.15.3"
}
DEBUG: Resolved tag constraint(branch="renovate/react-helmet-6.x")
{
  "image": "renovate/node",
  "tagConstraint": "10.15.3",
  "tagVersioning": "npm",
  "tag": "10.15.3"
}
DEBUG: Fetching Docker image: renovate/node:10.15.3(branch="renovate/react-helmet-6.x")
DEBUG: Finished fetching Docker image(branch="renovate/react-helmet-6.x")
DEBUG: Executing command(branch="renovate/react-helmet-6.x")
{
  "command": "docker run --rm --name=renovate_node --label=renovate_child -v \"/mnt/renovate/gh/datadotworld/ui\":\"/mnt/renovate/gh/datadotworld/ui\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -v \"/home/ubuntu/.npmrc\":\"/home/ubuntu/.npmrc\" -e NPM_CONFIG_CACHE -e npm_config_store -w \"/mnt/renovate/gh/datadotworld/ui\" renovate/node:10.15.3 bash -l -c \"npm i -g yarn@1.9.4 && sed -i 's/ steps,/ steps.slice(0,1),/' /home/ubuntu/.npm-global/lib/node_modules/yarn/lib/cli.js && npm i -g lerna@2.7.1 && yarn install --ignore-scripts --ignore-engines --ignore-platform && lerna bootstrap --no-ci -- --ignore-scripts --ignore-engines --ignore-platform\""
}
DEBUG: lock file error(branch="renovate/react-helmet-6.x")
{
  "cmd": [
    "yarn install --ignore-scripts --ignore-engines --ignore-platform",
    "lerna bootstrap --no-ci -- --ignore-scripts --ignore-engines --ignore-platform"
  ],
  "err": {
    "killed": false,
    "code": 1,
    "signal": null,
    "cmd": "docker run --rm --name=renovate_node --label=renovate_child -v \"/mnt/renovate/gh/datadotworld/ui\":\"/mnt/renovate/gh/datadotworld/ui\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -v \"/home/ubuntu/.npmrc\":\"/home/ubuntu/.npmrc\" -e NPM_CONFIG_CACHE -e npm_config_store -w \"/mnt/renovate/gh/datadotworld/ui\" renovate/node:10.15.3 bash -l -c \"npm i -g yarn@1.9.4 && sed -i 's/ steps,/ steps.slice(0,1),/' /home/ubuntu/.npm-global/lib/node_modules/yarn/lib/cli.js && npm i -g lerna@2.7.1 && yarn install --ignore-scripts --ignore-engines --ignore-platform && lerna bootstrap --no-ci -- --ignore-scripts --ignore-engines --ignore-platform\"",
    "stdout": "/home/ubuntu/.npm-global/bin/yarn -> /home/ubuntu/.npm-global/lib/node_modules/yarn/bin/yarn.js\n/home/ubuntu/.npm-global/bin/yarnpkg -> /home/ubuntu/.npm-global/lib/node_modules/yarn/bin/yarn.js\n+ yarn@1.9.4\nadded 1 package in 1.441s\n/home/ubuntu/.npm-global/bin/lerna -> /home/ubuntu/.npm-global/lib/node_modules/lerna/bin/lerna.js\n\n> command-join@2.0.1 postinstall /home/ubuntu/.npm-global/lib/node_modules/lerna/node_modules/command-join\n> npx -p @seangenabe/tnx tnx || exit 0\n\n\n    Thanks for installing command-join!\n\n    If you like this package, be sure to star its repo,\n    and please consider donating:\n\n        https://**redacted**@2.7.1\nadded 311 packages from 170 contributors in 28.733s\nyarn install v1.9.4\n[1/4] Resolving packages...\nDone in 1.27s.\n",
    "stderr": "npx: installed 1 in 2.853s\nlerna info version 2.7.1\nlerna info Bootstrapping 11 packages\nlerna info lifecycle preinstall\nlerna info Installing external dependencies\nlerna info Symlinking packages and binaries\nlerna info lifecycle postinstall\nlerna info lifecycle prepublish\nlerna info lifecycle prepare\nlerna ERR! execute Error occured with '@datadotworld/dw-bootstrap' while running 'npm run prepare'\nlerna ERR! npm run prepare [@datadotworld/dw-bootstrap] Output from stdout:\n\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\n> grunt dist\n\n\nlerna ERR! npm run prepare [@datadotworld/dw-bootstrap] Output from stderr:\nsh: 1: grunt: not found\nnpm ERR! file sh\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno ENOENT\nnpm ERR! syscall spawn\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\nnpm ERR! spawn ENOENT\nnpm ERR! \nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\n\nlerna ERR! execute Error occured with '@datadotworld/dw-bootstrap' while running 'npm run prepare'\nlerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.\n{ Error: Command failed: npm run prepare\nsh: 1: grunt: not found\nnpm ERR! file sh\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno ENOENT\nnpm ERR! syscall spawn\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\nnpm ERR! spawn ENOENT\nnpm ERR! \nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\n\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\n> grunt dist\n\n\n    at Promise.all.then.arr (/home/ubuntu/.npm-global/lib/node_modules/lerna/node_modules/execa/index.js:236:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)\n  code: 1,\n  killed: false,\n  stdout:\n   '\\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\\n> grunt dist\\n\\n',\n  stderr:\n   'sh: 1: grunt: not found\\nnpm ERR! file sh\\nnpm ERR! code ELIFECYCLE\\nnpm ERR! errno ENOENT\\nnpm ERR! syscall spawn\\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\\nnpm ERR! spawn ENOENT\\nnpm ERR! \\nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\\n\\nnpm ERR! A complete log of this run can be found in:\\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\\n',\n  failed: true,\n  signal: null,\n  cmd: 'npm run prepare',\n  timedOut: false,\n  pkg:\n   Package {\n     _package:\n      { name: '@datadotworld/dw-bootstrap',\n        description:\n         'The most popular front-end framework for developing responsive, mobile first projects on the web.',\n        version: '3.3.36',\n        keywords: [Array],\n        dwBaseName: 'dw-bootstrap',\n        homepage: 'http://getbootstrap.com',\n        author: 'Twitter, Inc.',\n        scripts: [Object],\n        style: 'dist/css/bootstrap.css',\n        less: 'less/bootstrap.less',\n        main: './dist/js/npm',\n        repository: [Object],\n        bugs: [Object],\n        license: 'MIT',\n        devDependencies: [Object],\n        engines: [Object],\n        files: [Array],\n        jspm: [Object] },\n     _location:\n      '/mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap' },\n  exitCode: 1 }\n",
    "message": "Command failed: docker run --rm --name=renovate_node --label=renovate_child -v \"/mnt/renovate/gh/datadotworld/ui\":\"/mnt/renovate/gh/datadotworld/ui\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -v \"/home/ubuntu/.npmrc\":\"/home/ubuntu/.npmrc\" -e NPM_CONFIG_CACHE -e npm_config_store -w \"/mnt/renovate/gh/datadotworld/ui\" renovate/node:10.15.3 bash -l -c \"npm i -g yarn@1.9.4 && sed -i 's/ steps,/ steps.slice(0,1),/' /home/ubuntu/.npm-global/lib/node_modules/yarn/lib/cli.js && npm i -g lerna@2.7.1 && yarn install --ignore-scripts --ignore-engines --ignore-platform && lerna bootstrap --no-ci -- --ignore-scripts --ignore-engines --ignore-platform\"\nnpx: installed 1 in 2.853s\nlerna info version 2.7.1\nlerna info Bootstrapping 11 packages\nlerna info lifecycle preinstall\nlerna info Installing external dependencies\nlerna info Symlinking packages and binaries\nlerna info lifecycle postinstall\nlerna info lifecycle prepublish\nlerna info lifecycle prepare\nlerna ERR! execute Error occured with '@datadotworld/dw-bootstrap' while running 'npm run prepare'\nlerna ERR! npm run prepare [@datadotworld/dw-bootstrap] Output from stdout:\n\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\n> grunt dist\n\n\nlerna ERR! npm run prepare [@datadotworld/dw-bootstrap] Output from stderr:\nsh: 1: grunt: not found\nnpm ERR! file sh\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno ENOENT\nnpm ERR! syscall spawn\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\nnpm ERR! spawn ENOENT\nnpm ERR! \nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\n\nlerna ERR! execute Error occured with '@datadotworld/dw-bootstrap' while running 'npm run prepare'\nlerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.\n{ Error: Command failed: npm run prepare\nsh: 1: grunt: not found\nnpm ERR! file sh\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno ENOENT\nnpm ERR! syscall spawn\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\nnpm ERR! spawn ENOENT\nnpm ERR! \nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\n\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\n> grunt dist\n\n\n    at Promise.all.then.arr (/home/ubuntu/.npm-global/lib/node_modules/lerna/node_modules/execa/index.js:236:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)\n  code: 1,\n  killed: false,\n  stdout:\n   '\\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\\n> grunt dist\\n\\n',\n  stderr:\n   'sh: 1: grunt: not found\\nnpm ERR! file sh\\nnpm ERR! code ELIFECYCLE\\nnpm ERR! errno ENOENT\\nnpm ERR! syscall spawn\\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\\nnpm ERR! spawn ENOENT\\nnpm ERR! \\nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\\n\\nnpm ERR! A complete log of this run can be found in:\\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\\n',\n  failed: true,\n  signal: null,\n  cmd: 'npm run prepare',\n  timedOut: false,\n  pkg:\n   Package {\n     _package:\n      { name: '@datadotworld/dw-bootstrap',\n        description:\n         'The most popular front-end framework for developing responsive, mobile first projects on the web.',\n        version: '3.3.36',\n        keywords: [Array],\n        dwBaseName: 'dw-bootstrap',\n        homepage: 'http://getbootstrap.com',\n        author: 'Twitter, Inc.',\n        scripts: [Object],\n        style: 'dist/css/bootstrap.css',\n        less: 'less/bootstrap.less',\n        main: './dist/js/npm',\n        repository: [Object],\n        bugs: [Object],\n        license: 'MIT',\n        devDependencies: [Object],\n        engines: [Object],\n        files: [Array],\n        jspm: [Object] },\n     _location:\n      '/mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap' },\n  exitCode: 1 }\n",
    "stack": "Error: Command failed: docker run --rm --name=renovate_node --label=renovate_child -v \"/mnt/renovate/gh/datadotworld/ui\":\"/mnt/renovate/gh/datadotworld/ui\" -v \"/tmp/renovate-cache\":\"/tmp/renovate-cache\" -v \"/home/ubuntu/.npmrc\":\"/home/ubuntu/.npmrc\" -e NPM_CONFIG_CACHE -e npm_config_store -w \"/mnt/renovate/gh/datadotworld/ui\" renovate/node:10.15.3 bash -l -c \"npm i -g yarn@1.9.4 && sed -i 's/ steps,/ steps.slice(0,1),/' /home/ubuntu/.npm-global/lib/node_modules/yarn/lib/cli.js && npm i -g lerna@2.7.1 && yarn install --ignore-scripts --ignore-engines --ignore-platform && lerna bootstrap --no-ci -- --ignore-scripts --ignore-engines --ignore-platform\"\nnpx: installed 1 in 2.853s\nlerna info version 2.7.1\nlerna info Bootstrapping 11 packages\nlerna info lifecycle preinstall\nlerna info Installing external dependencies\nlerna info Symlinking packages and binaries\nlerna info lifecycle postinstall\nlerna info lifecycle prepublish\nlerna info lifecycle prepare\nlerna ERR! execute Error occured with '@datadotworld/dw-bootstrap' while running 'npm run prepare'\nlerna ERR! npm run prepare [@datadotworld/dw-bootstrap] Output from stdout:\n\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\n> grunt dist\n\n\nlerna ERR! npm run prepare [@datadotworld/dw-bootstrap] Output from stderr:\nsh: 1: grunt: not found\nnpm ERR! file sh\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno ENOENT\nnpm ERR! syscall spawn\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\nnpm ERR! spawn ENOENT\nnpm ERR! \nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\n\nlerna ERR! execute Error occured with '@datadotworld/dw-bootstrap' while running 'npm run prepare'\nlerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.\n{ Error: Command failed: npm run prepare\nsh: 1: grunt: not found\nnpm ERR! file sh\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno ENOENT\nnpm ERR! syscall spawn\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\nnpm ERR! spawn ENOENT\nnpm ERR! \nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\n\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\n> grunt dist\n\n\n    at Promise.all.then.arr (/home/ubuntu/.npm-global/lib/node_modules/lerna/node_modules/execa/index.js:236:11)\n    at process._tickCallback (internal/process/next_tick.js:68:7)\n  code: 1,\n  killed: false,\n  stdout:\n   '\\n> @datadotworld/dw-bootstrap@3.3.36 prepare /mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap\\n> grunt dist\\n\\n',\n  stderr:\n   'sh: 1: grunt: not found\\nnpm ERR! file sh\\nnpm ERR! code ELIFECYCLE\\nnpm ERR! errno ENOENT\\nnpm ERR! syscall spawn\\nnpm ERR! @datadotworld/dw-bootstrap@3.3.36 prepare: `grunt dist`\\nnpm ERR! spawn ENOENT\\nnpm ERR! \\nnpm ERR! Failed at the @datadotworld/dw-bootstrap@3.3.36 prepare script.\\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\\n\\nnpm ERR! A complete log of this run can be found in:\\nnpm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-08T21_10_03_599Z-debug.log\\n',\n  failed: true,\n  signal: null,\n  cmd: 'npm run prepare',\n  timedOut: false,\n  pkg:\n   Package {\n     _package:\n      { name: '@datadotworld/dw-bootstrap',\n        description:\n         'The most popular front-end framework for developing responsive, mobile first projects on the web.',\n        version: '3.3.36',\n        keywords: [Array],\n        dwBaseName: 'dw-bootstrap',\n        homepage: 'http://getbootstrap.com',\n        author: 'Twitter, Inc.',\n        scripts: [Object],\n        style: 'dist/css/bootstrap.css',\n        less: 'less/bootstrap.less',\n        main: './dist/js/npm',\n        repository: [Object],\n        bugs: [Object],\n        license: 'MIT',\n        devDependencies: [Object],\n        engines: [Object],\n        files: [Array],\n        jspm: [Object] },\n     _location:\n      '/mnt/renovate/gh/datadotworld/ui/css-components/dw-bootstrap' },\n  exitCode: 1 }\n\n    at ChildProcess.exithandler (child_process.js:303:12)\n    at ChildProcess.emit (events.js:311:20)\n    at ChildProcess.EventEmitter.emit (domain.js:482:12)\n    at maybeClose (internal/child_process.js:1021:16)\n    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)"
  },
  "type": "lerna",
  "lernaClient": "yarn"
}
DEBUG: No updated lock files in branch(branch="renovate/react-helmet-6.x")
DEBUG: Branch timestamp: 2020-06-08T04:59:07.232Z(branch="renovate/react-helmet-6.x")
DEBUG: PR is less than a day old - raise error instead of PR(branch="renovate/react-helmet-6.x")
DEBUG: Passing lockfile-error up(branch="renovate/react-helmet-6.x")
INFO: Lock file error - aborting
```

**To Reproduce**

I do not own our renovatebot configuration, so I am not sure exactly how to set up a similar repository.

I have searched elsewhere for similar issues but have not found any.

**Additional context**

I looked around at various renovate docker repos (`node`, `npm`, `buildpack`) but have not found any recent changes that seem like they would cause `grunt` to not be available.

Hey, 

I do get these errors as well.
I can confirm that I still have the same issue with lerna bootstrap

npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated mkdirp-promise@5.0.1: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
lerna notice cli v3.22.1
lerna info Bootstrapping 3 packages
lerna info Installing external dependencies
lerna info Symlinking packages and binaries
lerna info lifecycle json-serverless-lib@1.5.42~prepare: json-serverless-lib@1.5.42
sh: 1: tsc: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! json-serverless-lib@1.5.42 compile: `tsc -p .`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the json-serverless-lib@1.5.42 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /tmp/renovate-cache/others/npm/_logs/2020-06-19T14_56_17_256Z-debug.log
lerna info lifecycle json-serverless-lib@1.5.42~prepare: Failed to exec prepare script
lerna ERR! lifecycle "prepare" errored in "json-serverless-lib", exiting 1


here`s my script config:

```javascript
  "scripts": {
    "test": "npx jest",
    "debug": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec node --inspect-brk -r ts-node/register example/index.ts",
    "start": "nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec 'ts-node' example/index.ts",
    "type-check": "tsc --noEmit",
    "type-check:watch": "npm run type-check -- --watch",
    "build": "tsc -b",
    "prepare": "npm run compile",
    "pretest": "npm run compile",
    "posttest": "npm run check",
    "check": "gts check",
    "clean": "gts clean",
    "compile": "tsc -p .",
    "fix": "gts fix"
  }, 
```

--- a/lib/manager/npm/post-update/lerna.ts
+++ b/lib/manager/npm/post-update/lerna.ts
@@ -55,8 +55,10 @@ export async function generateLockFiles(
       logger.warn({ lernaClient }, 'Unknown lernaClient');
       return { error: false };
     }
+    let lernaCommand = `lerna bootstrap --no-ci --ignore-scripts -- ${cmdOptions}`;
     if (global.trustLevel === 'high' && config.ignoreScripts !== false) {
       cmdOptions = cmdOptions.replace('--ignore-scripts ', '');
+      lernaCommand = lernaCommand.replace('--ignore-scripts ', '');
     }
     const tagConstraint = await getNodeConstraint(config);
     const execOptions: ExecOptions = {
@@ -93,7 +95,7 @@ export async function generateLockFiles(
     }
     logger.debug('Using lerna version ' + lernaVersion);
     preCommands.push(`npm i -g lerna@${quote(lernaVersion)}`);
-    cmd.push(`lerna bootstrap --no-ci -- ${cmdOptions}`);
+    cmd.push(lernaCommand);
     await exec(cmd, execOptions);
   } catch (err) /* istanbul ignore next */ {
     logger.debug(

--- a/lib/manager/npm/post-update/lerna.ts
+++ b/lib/manager/npm/post-update/lerna.ts
@@ -55,11 +55,12 @@ export async function generateLockFiles(
       logger.warn({ lernaClient }, 'Unknown lernaClient');
       return { error: false };
     }
-    let lernaCommand = `lerna bootstrap --no-ci --ignore-scripts -- ${cmdOptions}`;
+    let lernaCommand = `lerna bootstrap --no-ci --ignore-scripts -- `;
     if (global.trustLevel === 'high' && config.ignoreScripts !== false) {
       cmdOptions = cmdOptions.replace('--ignore-scripts ', '');
       lernaCommand = lernaCommand.replace('--ignore-scripts ', '');
     }
+    lernaCommand += cmdOptions;
     const tagConstraint = await getNodeConstraint(config);
     const execOptions: ExecOptions = {
       cwd,

